cdap-app-fabric/src/main/java/co/cask/cdap/operations/OperationalStatsService.java

```diff
@@ -170,7 +170,7 @@ protected void shutDown() throws Exception {
       try {
         mbs.unregisterMBean(objectName);
       } catch (InstanceNotFoundException e) {
-        LOG.debug("MBean {} not found while un-registering. Ignoring.", objectName);
+        LOG.warn("MBean {} not found while un-registering. Ignoring.", objectName);
```

Comment:
hmmm, not sure we need this. this is while trying to clean up. it shouldn't happen, but if it does, its ok to ignore.